dialogue-core/src/main/java/com/palantir/dialogue/core/ActiveRequestInstrumentationChannel.java

```diff
@@ -35,13 +36,13 @@
             Channel delegate,
             String channelName,
             final @CompileTimeConstant String stage,
-            DialogueClientMetrics metrics) {
+            TaggedMetricRegistry metrics) {
```

Comment:
I think we should prefer passing the Metrics wrapper instead of full metric registry to be more explicit about what metrics are produce by what components

Reply:
if we had a single DialogueMetrics class then I'd agree, but i really don't like keeping around so many different FooMetrics BarMetrics BazMetrics classes, and I think it makes the plumbing worse.

realistically, we're not in danger of adding any non-metric-schema thngs.